plugins/inputs/cloudwatch_metric_streams/cloudwatch_metric_streams.go

```diff
@@ -39,7 +39,7 @@ type CloudWatchMetricStreams struct {
 	ReadTimeout      config.Duration `toml:"read_timeout"`
 	WriteTimeout     config.Duration `toml:"write_timeout"`
 	AccessKey        string          `toml:"access_key"`
-	APICompatability bool            `toml:"api_compatability"`
+	APICompatibility bool            `toml:"api_compatibility"`
```

Comment:
We cannot do this change as this breaks existing configs! Please deprecate the old option and add a new one...